pkg/systemd/overview-cards/configurationCard.jsx

```diff
@@ -675,9 +675,9 @@ PageSystemInformationChangeSystime.prototype = {
 
         self.ntp_type = self.server_time.timedate.NTP ? (self.custom_ntp_enabled ? 'ntp_time_custom' : 'ntp_time') : 'manual_time';
         $('#change_systime [value="ntp_time"]')
-                .toggleClass("disabled", !self.server_time.timedate.CanNTP);
+                .prop("disabled", !self.server_time.timedate.CanNTP);
```

Comment:
I think this breaks: https://github.com/cockpit-project/cockpit/blob/master/pkg/systemd/overview-cards/configurationCard.jsx#L618-L621

Is there any test covering this functionality?

Reply:
> Is there any test covering this functionality?

I looked quickly and cannot see it where we would check that the button is disabled. If you would write the test, that would be great :)

Reply:
I am not opposed to writing the test, but I would need some pointers, i.e. how do I get "CanNTP" to false? I am not too familiar with Dbus and such.

Reply:
Ok, so this is broken and needs some attention.
Firstly, as you pointed out, it breaks that you can actually click on it. (use `:disabled` instead of `'.disabled'`)
Secondly, it does not change style and cursor. This style is again based on class `.disabled`. I don't know how you can actually trigger `CanNTP` to be false, but for testing, you can just use `.prop("disabled", true);` here.

Reply:
Yeah, for manual testing that makes sense, of course.

Reply:
Double checking, this is a `<li>` element, i'll just revert back to using the class.